build_runner/test/common/matchers.dart

```diff
@@ -30,6 +30,10 @@ class _AssetGraphMatcher extends Matcher {
     if (graph.allNodes.length != _expected.allNodes.length) matches = false;
     for (var node in graph.allNodes) {
       var expectedNode = _expected.get(node.id);
+      if (node.isDeleted != node.isDeleted) {
+        matchState['IsDeleted'] = [node.isDeleted, expectedNode.isDeleted];
```

Comment:
the string here is the description and should include the asset id, something like `IsDeleted of ${node.id}`

Reply:
Done.